airbnb/react-native-maps
# [zIndex]  Add support for setting zIndex on markers

Add support for setting the z-index on markers via the style.

Example usage:
```
<MapView.Marker
  style={styles.importantMarker}
  ... 
/>
```

```
const styles = StyleSheet.create({
  importantMarker: {
    zIndex: 1,
  },
});
```

Test plan: 
* Verify on both iOS and Android that with this fix any markers with a zIndex > 0 show up above any other markers.
* Verify without the change sometimes markers appear below others.
